lib/collective-sections.js

```diff
@@ -319,3 +319,77 @@ export const getFilteredSectionsForCollective = (collective, isAdmin, isHostAdmi
     return filterSectionsByData(sections, collective, isAdmin, isHostAdmin);
   }
 };
+
+export const NAVBAR_CATEGORIES = {
+  ABOUT: 'ABOUT',
+  BUDGET: 'BUDGET',
+  CONNECT: 'CONNECT',
+  CONTRIBUTE: 'CONTRIBUTE',
+  OTHER_INITIATIVES: 'OTHER_INITIATIVES', // Events, projects, connected collectives
+};
+
+/**
+ * Map sections to their categories. Any section that's not in this object will be considered
+ * as a "Widget" (aka. a section without navbar category).
+ */
+const SECTIONS_CATEGORIES = {
+  // About
+  [Sections.ABOUT]: NAVBAR_CATEGORIES.ABOUT,
+  [Sections.LOCATION]: NAVBAR_CATEGORIES.ABOUT,
+  // Connect
+  [Sections.CONVERSATIONS]: NAVBAR_CATEGORIES.CONNECT,
+  [Sections.UPDATES]: NAVBAR_CATEGORIES.CONNECT,
+  // Contribute
+  [Sections.TICKETS]: NAVBAR_CATEGORIES.CONTRIBUTE,
+  [Sections.CONTRIBUTE]: NAVBAR_CATEGORIES.CONTRIBUTE,
+  [Sections.CONTRIBUTORS]: NAVBAR_CATEGORIES.CONTRIBUTE,
+  [Sections.CONTRIBUTIONS]: NAVBAR_CATEGORIES.CONTRIBUTE,
+  [Sections.RECURRING_CONTRIBUTIONS]: NAVBAR_CATEGORIES.CONTRIBUTE,
+  [Sections.CONTRIBUTORS]: NAVBAR_CATEGORIES.CONTRIBUTE,
+  // Budget
+  [Sections.BUDGET]: NAVBAR_CATEGORIES.BUDGET,
+  // Events/Projects
+  [Sections.EVENTS]: NAVBAR_CATEGORIES.OTHER_INITIATIVES,
+  [Sections.PROJECTS]: NAVBAR_CATEGORIES.OTHER_INITIATIVES,
+};
+
+const WIDGET_SECTIONS = [Sections.GOALS]; // TODO: top contributors
+
+const convertSectionToNewFormat = ({ section, isEnabled, restrictedTo = null }) => ({
+  type: 'SECTION',
+  name: section,
+  isEnabled,
+  restrictedTo,
+});
+
+/**
+ * Converts legacy sections to their new format
+ */
+export const convertSectionsToNewFormat = sections => {
```

Comment:
And where will we use this? In `getFilteredSectionsForCollective`?

Reply:
Yes, the places were we use `getFilteredSectionsForCollective` should be able to handle both sections format while we transition